osmaxx-py/osmaxx/excerptexport/forms/excerpt_order_form_helpers.py

```diff
@@ -41,8 +48,8 @@ def get_existing_excerpt_choices_shortcut(user):
         ('Other excerpts [{count}]'.format(count=other_public(user).count()),
          tuple((excerpt.id, excerpt.name) for excerpt in other_public(user))
          ),
-        ('Countries [{count}]'.format(count=countries().count()),
-         tuple((excerpt.id, excerpt.name) for excerpt in countries())
+        ('Countries [{count}]'.format(count=len(get_country_choices())),
+         get_country_choices()
```

Comment:
Even if caching is involved, I'd feel better if `get_country_choices()` got only called once per execution of `get_existing_excerpt_choices_shortcut()`. Extract a local variable.